docs/docs/user/migration-guide/v4-v5.md

```diff
@@ -0,0 +1,22 @@
+# Migrate from V4 to V5
+
+## [Allow custom timeout for external services](https://github.com/bakdata/kpops/pull/485)
+
+The global `timeout` setting has been removed. Instead, an individual timeout can be set for each external service.
+
+#### config.yaml
+
+```diff
+- timeout: 300
+
+  kafka_rest:
+    url: "http://my-custom-rest.url:8082"
++   timeout: 300
+  kafka_connect:
+    url: "http://my-custom-connect.url:8083"
++   timeout: 300
+  schema_registry:
+    enabled: true
+    url: "http://my-custom-sr.url:8081"
++   timeout: 300
```

Comment:
I would mention that the default for each operation is 30 seconds, and I would also apply it to the diff. We moved away from the 300 😄